hw/ip/flash_ctrl/data/flash_ctrl.hjson.tpl

```diff
@@ -1093,6 +1099,51 @@
         ]
       },
 
+      { name: "STD_FAULT_STATUS",
+        desc: '''
+          Flash standard fault status register.
+          This register tabulates standard fault status of the flash.
+
+          These represent errors that occur in the standard structures of the design.
+          For example fsm integrity, couter integrity and tlul integrity.
```

Comment:
Is that a typo? Should it be "counter integrity"?

Reply:
yep will fix.